src/compat/posix/utime/Mybuild

```diff
@@ -0,0 +1,5 @@
+package embox.compat.posix
+
+static module utime {
+	source "utime.c"
```

Comment:
Please, add module dependency (for `kutime` function).

Reply:
added kutime in stub version, added depends of kutime in utime Mybuild